content/conf.py

```diff
@@ -34,6 +34,7 @@
     # githubpages just adds a .nojekyll file
     'sphinx.ext.githubpages',
     'sphinx_lesson',
+    'sphinx_tabs.tabs',
```

Comment:
This won't be needed with next version of sphinx-lesson, which will include this by default.  Working on releasing it now.
```suggestion
```
(don't apply suggest yet until it's released, though)

Reply:
sphinx-lesson 0.8.1 released, so I hope this is not necessary anymore...

Reply:
OK I will rebase code and throw that commit away.